IOMC/EventVertexGenerators/src/BetafuncEvtVtxGenerator.cc

```diff
@@ -140,3 +137,17 @@ void BetafuncEvtVtxGenerator::sigmaZ(double s) {
 }
 
 TMatrixD const* BetafuncEvtVtxGenerator::GetInvLorentzBoost() const { return &boost_; }
+
+void BetafuncEvtVtxGenerator::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  desc.add<double>("X0Incm", 0.0);
+  desc.add<double>("Y0Incm", 0.0);
+  desc.add<double>("Z0Incm", 0.0);
+  desc.add<double>("SigmaZIncm", 0.0);
+  desc.add<double>("BetaStarIncm", 0.0);
+  desc.add<double>("EmittanceIncm", 0.0);
+  desc.add<double>("TimeOffsetInns", 0.0);
+  desc.add<edm::InputTag>("src");
+  desc.add<bool>("readDB");
```

Comment:
Same question here.

Reply:
solved in https://github.com/cms-sw/cmssw/pull/43242#discussion_r1400567239